tools/accuracy_checker/setup.py

```diff
@@ -128,6 +131,6 @@ def prepare_requirements():
     install_requires=_requirements if not is_arm else '',
```

Comment:
do we still need all the magic with `is_arm` ? I suppose `requirements-core.in` are all available for arm

Reply:
we had some troubles with logic to search dependencies via setup_tools (it is starting to build from source), to prevent this behaviour on arm we use installing dependencies with pip, that is why not listed dependencies for setup

Reply:
let's try to remove and return back if issues still reproducible